packages/waci-pex/__tests__/agent.yml

```diff
@@ -0,0 +1,11 @@
+version: 3.0
```

Comment:
why do we have this file here? there should be an agent.yaml in the plugin folder itself

Reply:
We discussed and it was no more required in the source folder as the agent could be created without it ... I was finding it difficult to create the agent context for test. Hence, the file was added to only test.

Reply:
who is we?

Reply:
@BtencateSphereon and me